config/manifests/bases/codeflare-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: codeflare-operator.v0.0.0
```

Comment:
Should the version be v0.0.0, or should it start at v0.0.1 (like how we did our images)?

Reply:
Ha, 0.0.1 would be right. Will fix it